app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express")
+const app = express()
+app.use(express.json())
```

Comment:
You don't need this line yet. It only comes into play when dealing with POST/PATCH requests. Please delete it until then